helpers/pluck.js

```diff
@@ -1,8 +1,27 @@
 'use strict';
 
+const helpers = require('handlebars-helpers');
+
 const factory = () => {
     return function(collection, path) {
-        return collection.map(item => item[path]);
+        const isValidPath = typeof path === 'string';
```

Comment:
I would just return early for this rather than duplicating this condition multiple times below.

```
if (typeof path !== 'string') {
  return '';
}
```